docs/source/guides/parallel.rst

```diff
@@ -50,3 +49,7 @@ The dashboard requires an additional python package, bokeh, to work. Once bokeh
                                      n_jobs=2,
                                      dask_kwargs={'diagnostics_port': 8787}
                                      verbose=True)
+                                     
+Parallel Computation by Partioning Data
+-------------------------------
+As an alternative to Featuretool's parallelization, the data can be partitioned and run on multiple cores or a cluster using Dask or PySpark. For more information on partitioning the data and using Dask, see :doc:`/guides/performance`. An example of this approach can be seen in the `Featuretools on Dask notebook <https://github.com/Featuretools/Automated-Manual-Comparison/blob/master/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb>`_. Dask allows Featuretools to scale to multiple cores on a single machine or multiple machines on a cluster.
```

Comment:
just link to the other place in docs, no need to have link twice. that also means we once have to keep once place up to date.